### PR TITLE
SRS OC Gridded pipeline

### PR DESCRIPTION
**Pull request by lbesnard**
* add watch.d
 * create destination path script, + unittests and added to test.sh

### CODE REVIEW

File: SRS/srs_gridded_pipeline/oc_destination_path.sh [hunk 1]
```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# unit tests on shunit2_test_oc_destination_path.sh
+# returns the relative imos hierarchy of ocean colour netcdf file
+# author: laurent.besnard@utas.edu.au
+
+# remove replace nc4 extension with nc
+# $1 - netcdf filename
+_check_and_replace_netcdf_extension() {
+    local srs_file=$1; shift
+    echo $srs_file | sed "s/\.nc4$/.nc/g"
```
Comment (danfruehauf) [inline SRS/srs_gridded_pipeline/oc_destination_path.sh]: :+1: LOL

**Comment by danfruehauf** [comment]
All good. Waiting for the other PRs.

File: watch.d/SRS_GRIDDED_OC.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (danfruehauf) [inline watch.d/SRS_GRIDDED_OC.json]: Maybe we should call it `SRS_OC_GRIDDED` and also rename `SRS_GRIDDED_SST` to `SRS_SST_GRIDDED`.

Comment (lbesnard) [inline watch.d/SRS_GRIDDED_OC.json]: agree

Comment (danfruehauf) [inline watch.d/SRS_GRIDDED_OC.json]: Put a PR for that?

**Pull request merged**